specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHubCluster.json

```diff
@@ -0,0 +1,606 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "EventHubClusterManagementClient",
+    "description": "Azure Event Hubs Cluster client for managing Event Hubs Cluster resources.",
+    "version": "2018-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
PUT for namespaces can be found in our namespaces [Swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json) which has already been approved and is public. 

PUT for cluster is not supported in this API version because we do not yet support customers autonomously creating these resources. However, we do want to expose management APIs for the resources of this type which already exist or which our team builds for specific customers on demand. So we are releasing this preview API version without PUT supported for cluster.